Packs/Base/Scripts/CommonServerPython/CommonServerPython.py

```diff
@@ -9677,6 +9678,93 @@ def update_last_run_object(last_run, incidents, fetch_limit, start_fetch_time, e
     return last_run
 
 
+def send_events_to_xsiam(events: Union[str, list], vendor: str, product: str, data_format=None):
+    """
+    Send the fetched events into the XDR data-collector private api.
+
+    :type events: ``Union[str, list]``
+    :param events: The events to send to send to XSIAM server. Should be of the following:
+        1. List of strings or dicts where each string or dict represents an event.
+        2. String containing raw events separated by a new line.
```

Comment:
in which case events can be a string separated by a new line?
I think we better have a strict input type of list. Even better List[dict]

Reply:
In case the API returns a list of strings or just a long string with many lines representing logs, I don't want the developer to parse the logs on his own to a dict format.